ktor-client/ktor-client-android/jvm/test/io/ktor/client/engine/android/AndroidSpecificHttpsTest.kt

```diff
@@ -27,7 +27,7 @@ import kotlin.test.Test
 class AndroidSpecificHttpsTest : TestWithKtor() {
     override val server: EmbeddedServer<*, *> = embeddedServer(
         Netty,
-        applicationProperties {
+        rootConfig {
```

Comment:
If we're going to rename `application` to `server` at some point, `serverConfig` also looks like a good name.
@bjhham, @marychatte, what do you think?

Reply:
I thought about `serverConfig` before I saw your comment, so sound good for me

Reply:
If we're going to rename from `Application -> Server` at some point, then naming this `ServerConfig` would hit us with the dilemma of renaming the current `ApplicationConfig`.  I suppose we could go full circle and rename the current `ApplicationConfig` to `ServerProperties` at this point since it holds property references.

Reply:
Maybe I'll just go with ServerConfig for now since it seems most intuitive in the current scheme, and if there's future breaking changes then it doesn't matter so much since this is rarely used.